src/structure_new/src/linear_solver/4C_structure_new_solver_factory.cpp

```diff
@@ -259,8 +259,9 @@ std::shared_ptr<Core::LinAlg::Solver> Solid::SOLVER::Factory::build_meshtying_co
       if (sol_type == Inpar::CONTACT::solution_lagmult)
       {
         // provide null space information
-        if (prec == Core::LinearSolver::PreconditionerType::multigrid_muelu_contactsp)
-        { /* do nothing here */
+        if (prec == Core::LinearSolver::PreconditionerType::multigrid_muelu)
+        {
+          // do nothing
```

Comment:
Note to myself: I guess that both branches of this `if` statement should execute the same code. This is currently not the case, since AMG preconditioning for saddle-point formulations of contact systems have not been applied in the new time integration, yet. 

@PhilipOesterlePekrun is working in the old time integration, so he cannot fix this within this PR.